infra/tf/k8s_infra/main.tf

```diff
@@ -5,6 +5,11 @@ terraform {
 			source = "alekc/kubectl"
 			version = ">= 2.0.2"
 		}
+
+		betteruptime = {
+			source = "BetterStackHQ/better-uptime"
+			version = "0.4.0"
+		}
```

Comment:
Is this redundant?

Reply:
Yup, thanks for noticing. In fact, I think this is going to fix the current error I'm having while brining it up 😂